pandas/io/json/json.py

```diff
@@ -355,9 +355,9 @@ def read_json(
     path_or_buf : a valid JSON string or file-like, default: None
         The string could be a URL. Valid URL schemes include http, ftp, s3,
         gcs, and file. For file URLs, a host is expected. For instance, a local
-        file could be ``file://localhost/path/to/table.json``
+        file could be ``file://localhost/path/to/table.json``.
 
-    orient : string,
+    orient : str,
```

Comment:
```suggestion
    orient : str
```

Reply:
I think the preferred change, according to the docs, would be `str, optional`. However, the (many) other parameters in the docstring just put the default value. I will make it read `str, default None` for consistency's sake. I'll leave trudging through the function code and conforming the other parameter descriptions to the preferred style for another day . . .